src/main/java/uk/ac/cam/cl/dtg/isaac/api/managers/EventNotificationEmailManager.java

```diff
@@ -0,0 +1,175 @@
+                                         final PgScheduledEmailManager pgScheduledEmailManager) {
+        this.properties = properties;
+        this.contentManager = contentManager;
+        this.bookingManager = bookingManager;
+        this.userAccountManager = userAccountManager;
+        this.emailManager = emailManager;
+        this.pgScheduledEmailManager = pgScheduledEmailManager;
+    }
+
+    public void sendBookingStatusFilteredEmailForEvent(IsaacEventPageDTO event, String templateId, List<BookingStatus> bookingStatuses) throws SegueDatabaseException {
+        List<DetailedEventBookingDTO> eventBookings = bookingManager.adminGetBookingsByEventId(event.getId());
+        List<Long> ids = eventBookings.stream()
+                            .filter(DetailedEventBookingDTO -> bookingStatuses == null || bookingStatuses.contains(DetailedEventBookingDTO.getBookingStatus()))
+                            .map(DetailedEventBookingDTO::getUserBooked)
+                            .map(UserSummaryDTO::getId)
+                            .distinct().collect(Collectors.toList());
+        for (Long id : ids) {
+            try {
+                RegisteredUserDTO user = userAccountManager.getUserDTOById(id);
+                emailManager.sendTemplatedEmailToUser(user,
+                    emailManager.getEmailTemplateDTO(templateId),
+                    new ImmutableMap.Builder<String, Object>()
+                        .put("event.emailEventDetails", event.getEmailEventDetails() == null ? "" : event.getEmailEventDetails())
+                        .put("event", event)
+                        .build(),
+                    EmailType.SYSTEM);
+                log.debug(String.format("Sent email to user: %s %s, at: %s", user.getGivenName(), user.getFamilyName(), user.getEmail()));
+            } catch (NoUserException e) {
+                log.error(String.format("No user found with ID: %s", id));
+            } catch (ContentManagerException e) {
+                log.error("Failed to add the scheduled email sent time: ", e);
+            }
+        }
+    }
+
+    public void sendReminderEmails() {
+        // Magic number
+        Integer limit = 10000;
+        Integer startIndex = 0;
+        Map<String, List<String>> fieldsToMatch = Maps.newHashMap();
+        Map<String, Constants.SortOrder> sortInstructions = Maps.newHashMap();
+        Map<String, AbstractFilterInstruction> filterInstructions = Maps.newHashMap();
+        fieldsToMatch.put(TYPE_FIELDNAME, Collections.singletonList(EVENT_TYPE));
+        sortInstructions.put(DATE_FIELDNAME, Constants.SortOrder.DESC);
+        ZonedDateTime now = ZonedDateTime.now();
+        ZonedDateTime threeDaysAhead = now.plusDays(3);
+        DateRangeFilterInstruction
+            eventsWithinThreeDays = new DateRangeFilterInstruction(new Date(), Date.from(threeDaysAhead.toInstant()));
+        filterInstructions.put(ENDDATE_FIELDNAME, eventsWithinThreeDays);
```

Comment:
This is true, lots of this logic is copied from the events facade / methods, which use the enddate - it may be worth investigating if they actually work properly.